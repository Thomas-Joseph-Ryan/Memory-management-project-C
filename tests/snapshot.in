checkout 1
rollback 1
drop 1
set a 1 2 3
set b a
set c b
list entries
snapshot
set c 5 6 7
set d c a
list entries
snapshot
checkout 1
list entries
checkout 2
list entries
drop 2
checkout 2
list entries
set g 8 9
snapshot
list snapshots
rollback 1
list snapshots
checkout 3
list entries
pop c
list entries