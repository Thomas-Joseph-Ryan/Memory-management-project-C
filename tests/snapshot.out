> no such snapshot

> no such snapshot

> no such snapshot

> ok

> ok

> ok

> c [b]
b [a]
a [1 2 3]

> saved as snapshot 1

> ok

> ok

> d [c a]
c [5 6 7]
b [a]
a [1 2 3]

> saved as snapshot 2

> ok

> c [b]
b [a]
a [1 2 3]

> ok

> d [c a]
c [5 6 7]
b [a]
a [1 2 3]

> ok

> no such snapshot

> d [c a]
c [5 6 7]
b [a]
a [1 2 3]

> ok

> saved as snapshot 3

> 3
1

> ok

> 1

> no such snapshot

> c [b]
b [a]
a [1 2 3]

> b

> c []
b [a]
a [1 2 3]

> 
bye
